### PR TITLE
fixed dependencies for the alpha build

### PR DESCRIPTION
**Pull request by edewit**
Partially fixing issue AGCORDOVA-50

### CODE REVIEW

**Comment by edewit** [comment]
create a release after this has been merged 1.0.1

File: plugin.xml [hunk 1]
```diff
@@ -27,6 +27,8 @@
         compile 'org.jboss.aerogear:aerogear-android-core:2.0.0-alpha.1@aar'
         compile 'org.jboss.aerogear:aerogear-android-pipe:2.0.0-alpha.1@aar'
         compile 'org.jboss.aerogear:aerogear-android-store:2.0.0-alpha.1@aar'
+        compile 'com.google.code.gson:gson:1.7.2'
+        compile 'com.google.guava:guava:18.0'
```
Comment (matzew) [inline plugin.xml]: I thought we got rid of those? @danielpassos @secondsun

Comment (edewit) [inline plugin.xml]: only in 2.0.0 but that is not in maven central yet

Comment (matzew) [inline plugin.xml]: ah, ok. Instead of temporary adding this, perhaps we can wait a bit until 2.0.0 is available via maven central?

Comment (edewit) [inline plugin.xml]: I would rather have something that works in the mean time

Comment (matzew) [inline plugin.xml]: fair enough :-)

On Thu, Jan 15, 2015 at 11:28 AM, Erik Jan de Wit <notifications@github.com>
wrote:

> In plugin.xml
> <https://github.com/aerogear/aerogear-oauth2-cordova/pull/3#discussion-diff-23001022>
> :
>
> > @@ -27,6 +27,8 @@
> >          compile 'org.jboss.aerogear:aerogear-android-core:2.0.0-alpha.1@aar'
> >          compile 'org.jboss.aerogear:aerogear-android-pipe:2.0.0-alpha.1@aar'
> >          compile 'org.jboss.aerogear:aerogear-android-store:2.0.0-alpha.1@aar'
> > +        compile 'com.google.code.gson:gson:1.7.2'
> > +        compile 'com.google.guava:guava:18.0'
>
> I would rather have something that works in the mean time
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-oauth2-cordova/pull/3/files#r23001022>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

Comment (matzew) [inline plugin.xml]: :+1:

On Thu, Jan 15, 2015 at 11:31 AM, Matthias Wessendorf <matzew@apache.org>
wrote:

> fair enough :-)
>
> On Thu, Jan 15, 2015 at 11:28 AM, Erik Jan de Wit <
> notifications@github.com> wrote:
>
>> In plugin.xml
>> <https://github.com/aerogear/aerogear-oauth2-cordova/pull/3#discussion-diff-23001022>
>> :
>>
>> > @@ -27,6 +27,8 @@
>> >          compile 'org.jboss.aerogear:aerogear-android-core:2.0.0-alpha.1@aar'
>> >          compile 'org.jboss.aerogear:aerogear-android-pipe:2.0.0-alpha.1@aar'
>> >          compile 'org.jboss.aerogear:aerogear-android-store:2.0.0-alpha.1@aar'
>> > +        compile 'com.google.code.gson:gson:1.7.2'
>> > +        compile 'com.google.guava:guava:18.0'
>>
>> I would rather have something that works in the mean time
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/aerogear/aerogear-oauth2-cordova/pull/3/files#r23001022>
>> .
>>
>
>
>
> --
> Matthias Wessendorf
>
> blog: http://matthiaswessendorf.wordpress.com/
> sessions: http://www.slideshare.net/mwessendorf
> twitter: http://twitter.com/mwessendorf
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Pull request merged**